apps/test/unit/applab/ExporterTest.js

```diff
@@ -113,8 +112,13 @@ describe('Applab Exporter,', function() {
     server.respondWith('/blockly/media/third.jpg', 'blockly third.jpg content');
 
     // Needed to simulate fetch() response to '/projects/applab/fake_id/export_create_channel'
-    fakeFetch.install();
-    fakeFetch.respondWith(JSON.stringify({channel_id: 'new_fake_id'}));
+    sinon
+      .stub(window, 'fetch')
+      .returns(
+        Promise.resolve(
+          new Response(JSON.stringify({channel_id: 'new_fake_id'}))
+        )
+      );
```

Comment:
You might catch that we're being awfully nonspecific here - stubbing _every_ `fetch` call, not just those at the mentioned URL.

Turns out [this is exactly what `fake-fetch` was doing](https://github.com/msn0/fake-fetch/blob/master/index.js#L54).

Reply:
That seems acceptably specific for a unit test